uint8 ml_mode # determine which ML algorithm is used
string log_path #
bool rerun # whether plan from scratch or rerun previusly planned TMM, used for analysis to shorthen the time
---
trajectory_msgs/JointTrajectory[] ctamp_sol
uint32 n_samples # n_samples at the end of search
